Fitter/python/AnomalousCouplingEFTNegative.py

```diff
@@ -29,11 +29,15 @@ def __init__(self):
 
 
     def loadOperators(self,fpath):
-        print("Loading operators from {fpath}".format(fpath=fpath))
-        jsn = open(fpath,'r').read()
-        operators = json.loads(jsn)
-        self.alloperators = []
```

Comment:
Won't removing this case errors? Since `self.alloperators` is set below in a if block, it could be skipped. Plus, `self.alloperators.extend(operators[sig])` is called there, but we're not initializing `self.alloperators` as a list.